install_config/install/advanced_install.adoc

```diff
@@ -315,6 +315,30 @@ node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 ====
 
+Nodes only receive labels if you explicitly add them in the inventory using
+`*openshift_node_labels*` as described below. `*region=infra*` is the default
+selector. If you create nodes with that selector, then the router and registry
+will be installed.
+
+====
+----
+# default selectors for router and registry services
+# openshift_router_selector='region=infra'
+# openshift_registry_selector='region=infra'
+----
+====
```

Comment:
Thanks, @bfallonf and @brenton  I will leave the lines as is per Brenton's comment.